logstash-core/src/test/java/org/logstash/EventTest.java

```diff
@@ -3,12 +3,7 @@
 import java.io.IOException;
 import java.math.BigDecimal;
 import java.math.BigInteger;
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.Collection;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
+import java.util.*;
```

Comment:
nit: let's not mess up the style here :) This (`.*`) isn't allowed in the ES codestyle afaik.

Reply:
Hmmm, well, let me change my IDE settings then.